src/LazyOperations/ExponentialMap.jl

```diff
@@ -84,6 +84,24 @@ function size(spmexp::SparseMatrixExp, ax::Int)::Int
     return size(spmexp.M, ax)
 end
 
+function get_column(spmexp::SparseMatrixExp{N}, j::Int) where {N}
```

Comment:
I don't understand the purpose of having functions that just call `require`. Do you have a use case? Why do these functions not contain docstrings? Or the idea is that after doing the `using Expokit` this function is re-written with the one that actually contains code? In that case, i think that it is not a good idea. Just put the actual code just after the call to `require`, as we do with Polyhedra functions.

Reply:
> Or the idea is that after doing the using Expokit this function is re-written with the one that actually contains code?

Yes, that is the purpose. Currently if you do `get_column(E)` it just says that it does not know about that method. With this PR you get a nice error message that you should load `Expokit`.

> Put the actual code just after the call to require, as we do with Polyhedra functions.

That was the question above. I can try that, but it may not always work. If it works everywhere, we can remove `Requires` and I was not sure whether you want to go there. But fine, I also think this is the better thing to do.

Reply:
Done.